bigquery/tables.rb

```diff
@@ -72,10 +69,9 @@ def list_table_data project_id:, dataset_id:, table_id:
   # dataset_id = "ID of the dataset containing table"
   # table_id   = "ID of the table to display data for"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.